ipptool: add page (#12043)
tldr-pages/tldr: pages/common/ipptool.md

# ipptool

> > Send an IPP request and tests, displaying the results.

Reviewer: ```suggestion
> Send IPP requests for printer testing and troubleshooting.
```

How about something like this?

Author: Yeah,  the wording of your suggestion is better, but it seems `ipptool` is also for sending non-test IPP requests:

"ipptool is a command line tool for issuing IPP requests and receiving printer's/server's responses. ipptool is maintained as free software in the ippsample collection from the IEEE-ISTO Printer Working Group (PWG). ipptool is used for development and debugging of IPP-related software and for PWG self-certification for IPP Everywhere printers for driverless printing."

Reviewer: Maybe we can take it from the above description itself?

```suggestion
> Issue IPP requests and receive printer's/server's responses.
```
